odoo/addons/test_lint/tests/test_eslint.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+import subprocess
+from unittest import skipIf
+from odoo import tools
+
+from . import lint_case
+
+RULES = ('{'
+        'no-undef: error,'
+        'no-restricted-globals: ["error", "event", "self"]'
+        '}'
+)
+PARSER_OPTIONS = '{ecmaVersion: 2019, sourceType: module}'
+GLOBAL = ','.join([
+        'owl',
+        'odoo',
+        '$',
+        'jQuery',
+        '_',
+        'Chart',
+        'chrome',
```

Comment:
Most globals are self-explanatory but those which aren't like this one should probably have a comment. What is the `chrome` global, where does it come from?